### DISCUSSION TITLE
Failing train (botfront version 1.0.5)

### DISCUSSION THREAD

**Discussion by Abu-Ishak** [botfront/botfront#1004] [Q&A]
I have a new project installed on GCP VM. I've added some story and NLU examples, but when I want to train them is says as worning:
`
Error: An unexpected error occurred during training. Error: string indices must be integers
`
Here is my logs:
`
botfront-app |       upgradeOrConnect: false,
botfront-app |       parser: null,
botfront-app |       maxHeadersCount: null,
botfront-app |       reusedSocket: false,
botfront-app |       host: 'rasa',
botfront-app |       protocol: 'http:',
botfront-app |       _redirectable: [Writable],
botfront-app |       [Symbol(kCapture)]: false,
botfront-app |       [Symbol(kNeedDrain)]: false,
botfront-app |       [Symbol(corked)]: 0,
botfront-app |       [Symbol(kOutHeaders)]: [Object: null prototype]
botfront-app |     },
botfront-app |     data: <Buffer 7b 22 76 65 72 73 69 6f 6e 22 3a 22 32 2e 33 2e 33 22 2c 22 73 74 61 74 75 73 22 3a 22 66 61 69 6c 75 72 65 22 2c 22 6d 65 73 73 61 67 65 22 3a 22 41 ... 146 more bytes>
botfront-app |   },
botfront-app |   isAxiosError: true,
botfront-app |   toJSON: [Function: toJSON]
botfront-app | }
botfront-app | 2021-04-08T09:27:50.587Z [error] : Training project fptCJXB9ne5SuaauH - 101433.43 ms user: r6tZnyiGhZz9EoxYT /imports/api/instances/instances.methods.js - rasa.train with {"projectId":"fptCJXB9ne5SuaauH"}    error: Request failed with status code 500 at settle (/opt/bundle/bundle/programs/server/npm/node_modules/axios/lib/core/settle.js:17:12)
botfront-app | 2021-04-08T09:27:50.739Z [updated]: Marked trainning as stopped, User abuishak2018@gmail.com updated project fptCJXB9ne5SuaauH in project fptCJXB9ne5SuaauH - user: Abu Ishak abuishak2018@gmail.com - projectId: fptCJXB9ne5SuaauH - ressource id: fptCJXB9ne5SuaauH - ressource type: project - operation: project-updated -  before: {"projectBefore":{"_id":"fptCJXB9ne5SuaauH","disabled":true,"name":"Amez","namespace":"bf-amez","defaultLanguage":"en","defaultDomain":{"content":"slots:\n  disambiguation_message:\n    type: unfeaturized\nactions:\n  - action_botfront_disambiguation\n  - action_botfront_disambiguation_followup\n  - action_botfront_fallback\n  - action_botfront_mapping"},"chatWidgetSettings":{"title":"Amez","subtitle":"Happy to help","inputTextFieldHint":"Type your message...","initPayload":"/get_started","hideWhenNotConnected":true},"nluThreshold":0.75,"timezoneOffset":0,"enableSharing":false,"updatedAt":"2021-04-08T09:27:43.542Z","deploymentEnvironments":[],"storyGroups":["Gjhe5ht8T9Q9aZS9c","ChDYKB6KP4AF5KebA","Ggejcga6PHk86zCD6","RW5wWW9ZjxWAMqEDd"],"allowContextualQuestions":false,"languages":["en"],"training":{"instanceStatus":"training","startTime":"2021-04-08T09:26:09.108Z"}}} -  after: {"projectAfter":{"_id":"fptCJXB9ne5SuaauH","disabled":true,"name":"Amez","namespace":"bf-amez","defaultLanguage":"en","defaultDomain":{"content":"slots:\n  disambiguation_message:\n    type: unfeaturized\nactions:\n  - action_botfront_disambiguation\n  - action_botfront_disambiguation_followup\n  - action_botfront_fallback\n  - action_botfront_mapping"},"chatWidgetSettings":{"title":"Amez","subtitle":"Happy to help","inputTextFieldHint":"Type your message...","initPayload":"/get_started","hideWhenNotConnected":true},"nluThreshold":0.75,"timezoneOffset":0,"enableSharing":false,"updatedAt":"2021-04-08T09:27:50.595Z","deploymentEnvironments":[],"storyGroups":["Gjhe5ht8T9Q9aZS9c","ChDYKB6KP4AF5KebA","Ggejcga6PHk86zCD6","RW5wWW9ZjxWAMqEDd"],"allowContextualQuestions":false,"languages":["en"],"training":{"status":"failure","instanceStatus":"notTraining","endTime":"2021-04-08T09:27:50.592Z","message":"Request failed with status code 500 at settle (/opt/bundle/bundle/programs/server/npm/node_modules/axios/lib/core/settle.js:17:12)"}}} 

`
Any help would be appropriate.

**Comment by kpritzelhentley** [comment]
I have the same issue on a fresh quick installation on Ubuntu 20.04 VM. Would be great to get some help with this, don't know where to start at all...

botfront-rasa |     name = item["value"]
botfront-rasa | TypeError: string indices must be integers
botfront-rasa | 
botfront-rasa | 2021-04-09 05:59:44 ERROR    rasa.server  - An unexpected error occurred during training. Error: string indices must be integers
